ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -6,16 +6,49 @@ import {
 } from '@playwright/test';
 import { clickInTheMiddleOfTheScreen, pressButton } from '@utils/clicks';
 import { ELEMENT_TITLE } from './types';
-import { TopPanelButton, waitForRender } from '..';
+// import { DELAY_IN_SECONDS, TopPanelButton, waitForRender } from '..';
```

Comment:
delete all commented lines if they are not used in code or they don't give any information

Reply:
Done.